src/newsfeed/infrastructure/event_storages.py

```diff
@@ -179,7 +183,7 @@ def __init__(self, config: Dict[str, str]):
 
     async def delete_by_fqid(self, newsfeed_id: str, event_id: str) -> None:
         """Delete data of specified event."""
-        async with self._connection.get() as connection:
```

Comment:
I've realized that we can refactor next two lines

```python
        async with self._pool.get() as connection:
            redis = aioredis.commands.Redis(connection)
````

with an async context manager:

```python
from contextlib import asynccontextmanager

@asynccontextmanager
async def _get_connection():
    async with self._pool.get() as connection:
        redis = aioredis.commands.Redis(connection)
        yield redis
```

In that case we would not need to wrap pool connection into `aioredis.commands.Redis` every time, but just call `self._get_connection()`:

```python
        async with self._get_connection() as redis:
            ...
```

Reply:
Yeap, it looks cool! Thank you a lot!